components/shiphero/sources/order-allocated-instant/order-allocated-instant.mjs

```diff
@@ -0,0 +1,27 @@
+import common from "../common/webhook.mjs";
+import events from "../common/events.mjs";
+import sampleEmit from "./test-event.mjs";
+
+export default {
+  ...common,
+  key: "shiphero_order-allocated-instant",
+  name: "Order Allocated (Instant)",
+  description: "Emit new event when an order is allocated. [See the documentation](https://developer.shiphero.com/webhooks#webhook_create).",
+  type: "source",
+  version: "0.0.1",
+  dedupe: "unique",
+  methods: {
+    ...common.methods,
+    getEventName() {
+      return events.ORDER_ALLOCATED;
+    },
+    generateMeta(resource) {
+      return {
+        id: resource.id,
```

Comment:
Should this be `resource.order_id`? The test-event doesn't show `id` in the resource.
```suggestion
        id: resource.order_id,
```